src/conversions.jl

```diff
@@ -654,13 +612,13 @@ Prints all available gradient names.
 """
 function available_gradients()
     println("Gradient Symbol/Strings:")
-    for name in sort(collect(all_gradient_names))
+    for name in sort(collect(@eval PlotUtils.cgradients()))
```

Comment:
eval!?

Reply:
Should be no reason for that, the function just returns a vector of symbols

Reply:
Woops, forgot to push cleanup...will do that when I get back to my laptop.

Reply:
I'm a bit embarrassed I missed that

Reply:
I'd put that there as a test, it was removed in the cleanup commotion that I havent pushed yet ;)